SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -175,7 +175,8 @@ public static CaseNotesDocument ToEntity(this CreateCaseNoteRequest request)
             GenericCaseNote note = new GenericCaseNote()
             {
                 Timestamp = DateTime.Now.ToString("dd/MM/yyy hh:mm"),
-                DateOfBirth = request.DateOfBirth.ToString(),
+                DateOfBirth = request.DateOfBirth?.ToString(),
```

Comment:
Do we want to return the Datetime properties here as ISO format?

Reply:
We have aligned this with existing data i.e. used dd/mm/yyyy format